formspree/users/views.py

```diff
@@ -50,3 +54,33 @@ def logout():
 @login_required
 def dashboard():
     return render_template('users/dashboard.html', user=current_user)
+
+@login_required
+def forms():
+    if request.method == 'GET':
+        if request_wants_json():
+            return jsonerror(501, {'error': "This endpoint may return the list of forms for the logged user."})
+        else:
+            return redirect(url_form('dashboard'))
+
+    # Create a new form
+    if not current_user.upgraded:
+        return jsonerror(403, {'error': "Please upgrade your account."})
+
+    email = request.get_json().get('email') or abort(400)
+    if not IS_VALID_EMAIL(email):
+        return jsonerror(400, {'error': "The email you sent is not a valid email."})
+
+    form = Form(email, owner=current_user)
+    DB.session.add(form)
+    DB.session.commit()
+
+    # A unique identifier for the form that maps to its id,
+    # but doesn't seem like a sequential integer
+    random_like_string = form.get_random_like_string()
+
+    return jsonify({
+        'ok': True,
+        'random_like_string': random_like_string,
+        'submission_url': settings.API_ROOT + '/' + random_like_string
+    })
```

Comment:
I didn't know where to put it. But considering that the forms module is currently for form submissions and spontaneous form creation, and that this endpoint only works for logged and upgraded users (so that it is not just the "forms API", but the "forms-for-user API"), it made some sense to put it here.